crates/fluvio-cli/src/error.rs

```diff
@@ -117,57 +119,47 @@ impl CliError {
         }
     }
 
-    /// Looks at the error value and attempts to gracefully handle reporting it
+    /// Looks at the error value and attempts to create a user facing error message
     ///
     /// Sometimes, specific errors require specific user-facing error messages.
     /// Here is where we define those messages, as well as the exit code that the
     /// program should return when exiting after those errors.
-    pub fn print(self) -> Result<()> {
+    pub fn get_user_error(self) -> Result<String> {
```

Comment:
nit: it looks like this could be `&'static str`

Reply:
Ok

Reply:
Do you have any clue why the "format document" or `cargo fmt` does not work on error.rs?

Reply:
not sure to be honest